### ISSUE TITLE
[Feature]: Light Card Temperature Color Control Readout

### ISSUE THREAD

**Issue by GitHubGoody** [piitaya/lovelace-mushroom#391]
### Is your feature request related to a problem?

I cannot see the value of the temperature color while setting.

### Describe the solution you'd like

While sliding the temperature color, show the value of the temperature color slider.

As a bonus, it could also display when in range of the more typical Kelvin settings (e.g. warm white, cool white, daylight).

### Describe alternatives you've considered

None

### Additional context

https://blog.lifx.com/understanding-kelvin-and-light-temperature-5e898ed6dc90

**Comment by piitaya** [comment]
I think it can be good to display the value for all sliders. May be something like this ? https://ulic75.github.io/flat-slider/demos ?

**Comment by GitHubGoody** [comment]
> I think it can be good to display the value for all sliders. >

I agree. LIFX does this pretty well by displaying the color value as a 0-360° value where 0°=red, 120°=green, and 240°=blue.

I WOULD also like the option to add ~5 discrete color options...maybe most recent?

![image](https://user-images.githubusercontent.com/46235745/164976533-53e1c730-d336-4cab-8a63-1422cf0203e1.png)

>May be something like this ? https://ulic75.github.io/flat-slider/demos ?
>

I think the way the value is displayed while setting it looks good. Personally, I'd also like to see the currently set values for brightness and color/color temp (as configured) right under the card name as well.
